datetime/tests/time/mod.rs

```diff
@@ -72,22 +85,45 @@ fn run_test_time(
     assert!(stderr.contains(expected_error));
 }
 
+#[test]
+fn test_smoke_help() {
```

Comment:
This seems like a test that will break every time the command line usage is updated?  "fragile" test is sub-optimal.